source/Mission.cpp

```diff
@@ -177,6 +179,24 @@ void Mission::Load(const DataNode &node)
 						" They are now mission-level properties:");
 			}
 		}
+		else if(child.Token(0) == "outfit" && child.Size() >= 3)
+		{
+			outfitStr = child.Token(1);
+			outfitUnits = child.Value(2);
+			if(child.Size() >= 4)
+				outfitLimit = child.Value(3);
+			if(child.Size() >= 5)
+				outfitProb = child.Value(4);
+		}
+		else if(child.Token(0) == "outfitter" && child.Size() >= 3)
+		{
+			outfitterStr = child.Token(1);
+			outfitUnits = child.Value(2);
```

Comment:
if I'm reading this right you can't have outfit and outfitter in the same mission, since they use the same outfitUnits, limits etc so I'd say just add a warning if one of the two already was defined when the other one is read, to say we only support one of the two

Reply:
Can do

Reply:
do add that the first one still is overriten by the second one in the warning

Reply:
Warnings added closing out this covo